src/components/BulkEditAccessPoints/components/BulkEditAPTable/index.js

```diff
@@ -6,64 +6,36 @@ import { EditableRow } from './components/EditableRow';
 import { EditableCell } from './components/EditableCell';
 import styles from './index.module.scss';
 
-const BulkEditAPTable = ({
-  tableColumns,
-  tableData,
-  onEditedRows,
-  onLoadMore,
-  isLastPage,
-  resetEditedRows,
-}) => {
+const BulkEditAPTable = ({ tableColumns, tableData, onEditedRows, onLoadMore, isLastPage }) => {
   const [bulkEditTableData, setTableData] = useState([]);
   const [editedRows, setEditedRows] = useState([]);
+
+  useEffect(() => {
+    onEditedRows(editedRows);
+  }, [editedRows]);
+
```

Comment:
why are there two states of editedRows ? there's one here and one in the parent, why isn't it just one in the parent?